tests/cnf/core/network/cni/tests/sysctl/api.go

```
@@ -40,5 +42,99 @@ var _ = Describe("CNF Sysctl API", Ordered, Label(tsparams.LabelSysctlTestCases)
 			By("Wait until sysctl failed event")
 			waitUntilEventListContainsSysctlFailedCreatePodSandBoxMessage(tsparams.GlobalSysctlFlag)
 		})
+
+		It("one NAD, forward interface level duplicated flags", reportxml.ID("50346"), func() {
+			Skip("TC skipped due to BZ:2077683")
+
+			By("Define and create NAD with duplicated sysctl flag")
+			duplicatedFlag := createSysctlTuningNadWithDuplicatedKernelArg(
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)